modules/setting/repository.go

```diff
@@ -35,8 +35,12 @@ var (
 		AccessControlAllowOrigin                string
 		UseCompatSSHURI                         bool
 		DefaultCloseIssuesViaCommitsInAnyBranch bool
-		EnablePushCreateUser                    bool
-		EnablePushCreateOrg                     bool
+
+		EnableKanbanBoard           bool
+		ProjectBoardBasicKanbanType []string
+		ProjectBoardBugTriageType   []string
+		EnablePushCreateUser        bool
```

Comment:
would say rebase issue (code format)
 * dont remove **EnablePushCreateUser** and **EnablePushCreateOrg** to add it below KanBan settings